index.html

```diff
@@ -114,8 +104,9 @@ <h3>Terminology</h3>
   <p><dfn data-lt="language negotiation">Language negotiation</dfn> is any process which selects or filters content based on language. Usually this implies selecting content in a single language (or falling back to some meaningful default language that is available) by finding the best matching values when several languages or locales [[LTLI]] are present in the content. Some common language negotiation algorithms include the Lookup algorithm in [[BCP47]] or the BestFitMatcher in [[ECMA-402]].</p>
   <p><dfn>LTR</dfn> stands for "left-to-right" and refers to the inline base direction of left-to-right [[UAX9]]. This is the base text direction used by languages whose starting character progression begins on the left side of the page in horizontal text. It's used for scripts such as Latin, Cyrillic, Devanagari, and many others.</p>
   <p><dfn>RTL</dfn> stands for "right-to-left" and refers to the inline base direction of right-to-left [[UAX9]]. This is the base text direction used by languages whose starting character progression begins on the right side of the page in horizontal text. It's used for a variety of scripts which include Arabic, Hebrew, N'Ko, Adlam, Thaana, and Syriac among others.</p>
+<p><dfn>Bidirectional text</dfn> is a term used to refer to text that mixes runs of both LTR and RTL text inline. It is common for text written in Arabic, Hebrew, etc. to contain short runs of Latin text, and several of the scripts that are written predominantly right-to-left write their numbers from left-to-right. Bidirectional text of this kind is the source of many of the difficulties when dealing with RTL scripts.</p>
```

Comment:
This needs more work, I think.

* I would introduce "bidi" has a shorthand for this term. Or do you mean to use this term distinctly?
* _is a term_ is redundant in a definition
* the examples of Arabic/Hebrew, while amended with "etc", may be too specifically focused on those scripts. Also it's possible that they'll be read as meaning the Arabic/Hebrew languages
* _short runs of Latin text_ is overly specific. It's by far the most common case, of course. But we should allow for other inclusions.
* _bidirectional text of this kind_ reuses the term in a way that implies there are other kinds of bidirectional text.

Perhaps:

**Bidirectional text** (often referred to as "_bidi_" for short) refers to text that includes runs of both LTR and RTL text inline. It is common for right-to-left scripts, such as Arabic and Hebrew, to contain short runs of left-to-right text (most commonly in the Latin script), and several of the scripts that are predominantly right-to-left write their numbers from left-to-right. Bidirectional text is the source of many of the difficulties when dealing with RTL scripts.